internal/backends/qt/qt_accessible.rs

```diff
@@ -0,0 +1,699 @@
+        let item = self.item.clone();
+        let p = self.project_ref();
+        p.label_tracker.evaluate_as_dependency_root(move || {
+            if let Some(item_rc) = item.upgrade() {
+                item_rc.accessible_string_property(AccessibleStringProperty::Label);
+            }
+        });
+    }
+
+    fn arm_description_tracker(self: Pin<&Self>) {
+        let item = self.item.clone();
+        let p = self.project_ref();
+        p.description_tracker.evaluate_as_dependency_root(move || {
+            if let Some(item_rc) = item.upgrade() {
+                item_rc.accessible_string_property(AccessibleStringProperty::Description);
+            }
+        });
+    }
+
+    fn arm_focus_delegation_tracker(self: Pin<&Self>) {
+        let item = self.item.clone();
+        let p = self.project_ref();
+        p.focus_delegation_tracker.evaluate_as_dependency_root(move || {
+            if let Some(item_rc) = item.upgrade() {
+                item_rc.accessible_string_property(AccessibleStringProperty::DelegateFocus);
+            }
+        });
+    }
+}
+
+cpp! {{
+    #include <QtWidgets/QtWidgets>
+
+    #include <memory>
+
+    /// KEEP IN SYNC WITH CONSTANTS IN RUST!
+    const uint32_t NAME { QAccessible::Name };
+    const uint32_t DESCRIPTION { QAccessible::Description };
+    const uint32_t VALUE { QAccessible::Value};
+    const uint32_t CHECKED { QAccessible::UserText };
+    const uint32_t VALUE_MINIMUM { CHECKED + 1 };
+    const uint32_t VALUE_MAXIMUM { VALUE_MINIMUM + 1 };
+    const uint32_t VALUE_STEP { VALUE_MAXIMUM + 1 };
+
+    // ------------------------------------------------------------------------------
+    // Helper:
+    // ------------------------------------------------------------------------------
+
+    class Descendents {
+    public:
+        Descendents(void *root_item) {
+            rustDescendents = rust!(Descendents_ctor [root_item: *mut c_void as "void*"] ->
+                    SharedVector<ItemRc> as "void*" {
+                let mut descendents = i_slint_core::accessibility::accessible_descendents(
+                        &*(root_item as *mut ItemRc));
+                SharedVector::from_slice(&descendents)
+            });
+        }
+
+        size_t count() {
+            return rust!(Descendents_count [rustDescendents: SharedVector<ItemRc> as "void*"] -> usize as "size_t" {
+               rustDescendents.len()
+            });
+        }
+
+        void* itemAt(size_t index) {
+            return rust!(Descendents_itemAt [rustDescendents: SharedVector<ItemRc> as "void*",
+                                             index: usize as "size_t"]
+                    -> *mut c_void as "void*" {
+                let item_rc = rustDescendents[index].clone();
+                let mut item_weak = Box::new(item_rc.downgrade());
+
+                let result = core::ptr::addr_of_mut!(*item_weak);
+
+                std::mem::forget(item_weak);
+
+                result as _
```

Comment:
is that not simply  `Box::into_raw(item_weak)`

Reply:
Now that you point it out: Probably yes:-)